docs/source/configuration.md

```diff
@@ -29,3 +29,20 @@ jupyter lab --YDocExtension.document_cleanup_delay=100
 # The YStore class to use for storing Y updates (default: JupyterSQLiteYStore).
 jupyter lab --YDocExtension.ystore_class=pycrdt_websocket.ystore.TempFileYStore
 ```
+
+There is an experimental feature that is currently only supported by the
+[Jupyverse](https://github.com/jupyter-server/jupyverse) server
+(not yet with [jupyter-server](https://github.com/jupyter-server/jupyter_server)):
```

Comment:
Is there an issue with proposal to implement it in jupyter-server to allow users to track the progress (or offer to contribute)?

Was implementing this in jupyter-server previously discussed in jupyter-server calls?

Reply:
There is https://github.com/jupyter-server/jupyter_server/issues/900, and I think @Zsailer is interested in implementing it.